lib/gamnit/depth/depth_core.nit

```diff
@@ -0,0 +1,233 @@
+# This file is part of NIT ( http://www.nitlanguage.org ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Base entities of the depth 3D game framework
+module depth_core
+
+intrude import gamnit::flat
+
+# Instance of a `Model` in the game world
+class Actor
+
+	# Model used to dray this actor
+	var model: Model
+
+	# Position of this sprite in world coordinates
+	var center: Point3d[Float] is writable
+
+	# Rotation on the Z axis
+	var rotation = 0.0 is writable
+
+	# Scale applied to this sprite
+	var scale = 1.0 is writable
+
+	# Transparency applied to the texture on draw
+	var alpha = 1.0 is writable
+end
+
+# Full model
```

Comment:
full model of what? I have a hard time to understand what is/isn't/should be in a depthmnit model